src/SmartComponents/PackageDetail/PackageDetail.test.js

```diff
@@ -60,7 +61,12 @@ afterEach(() => {
 
 describe('PackageDetail.js', () => {
     it('should match the snapshot', ()  => {
-        expect(toJson(wrapper)).toMatchSnapshot();
+        const { asFragment } = render(
```

Comment:
I think we should change this test and instead of taking a snapshot, verify via a RTL query that a title for the loaded package (or some other uniq property/element) exists on screen.

Reply:
Added 2 tests that checks the component text